### DISCUSSION TITLE
有人知道input的json要填什麼嗎

### DISCUSSION THREAD

**Discussion by 123hi123** [hect0x7/JMComic-Crawler-Python#269] [Q&A]
![image](https://github.com/user-attachments/assets/85f5b880-b403-412c-9df8-340505c82611)
為什麼會說是json是因為如果我隨便輸入一段數字的時候會顯示不合法的json格式

**Comment by hect0x7** [comment]
完全没见过这个页面，为啥不用github网页

**Reply by 123hi123** [reply]
這是蘋果的捷徑 我才應該只要知道json的格式就可以也 不是很急 就好奇能不能用看看

**Reply by hect0x7** [reply]
> 這是蘋果的捷徑 我才應該只要知道json的格式就可以也 不是很急 就好奇能不能用看看

不懂诶，感觉不太“捷徑”😂

**Reply by 123hi123** [reply]
你說對了 還要額外研究 確實不太行
